packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -43,6 +52,12 @@ const Redirect = observer(() => {
         window.location.reload();
     }
 
+    const liveChat = useLiveChat(false, loginid);
```

Comment:
@shahzaib-deriv Maybe we need to add user's data to the chat if the user logged in? For example as we did in `LiveChat` component:

`const liveChat = useLiveChat(has_cookie_account, loginid);`